html-templates/includes/site.js-top.tpl

```diff
@@ -0,0 +1 @@
+<script src='https://www.google.com/recaptcha/api.js'></script>
```

Comment:
@RowdyChildren Do we need this to be loaded on every page on the site rather than just on the register page?